subprojects/core/src/main/java/org/gradle/internal/vfs/EventDrivenFileWatcherRegistry.java

```diff
@@ -28,31 +27,30 @@
 import java.io.IOException;
 import java.nio.file.Paths;
 import java.util.Optional;
-import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.Function;
 
 import static org.gradle.internal.vfs.watch.FileWatcherRegistry.Type.CREATED;
 import static org.gradle.internal.vfs.watch.FileWatcherRegistry.Type.INVALIDATED;
 import static org.gradle.internal.vfs.watch.FileWatcherRegistry.Type.MODIFIED;
 import static org.gradle.internal.vfs.watch.FileWatcherRegistry.Type.REMOVED;
 
-public abstract class AbstractEventDrivenFileWatcherRegistry implements FileWatcherRegistry {
-    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractEventDrivenFileWatcherRegistry.class);
+public class EventDrivenFileWatcherRegistry implements FileWatcherRegistry {
```

Comment:
Why not `DefaultFileWatcherRegistry`? Do we expect more implementations of this?

Reply:
`DefaultFileWatcherRegistry` it is.